tests/helpers/preload-assets.js

```diff
@@ -0,0 +1,26 @@
+import Ember from 'ember';
+import AssetLoader from 'ember-asset-loader/services/asset-loader';
```

Comment:
I actually think this will move to the `ember-asset-loader` repo. But duly noted, will probably use `addon-test-support`.